### ISSUE TITLE
New SNP Track

### ISSUE THREAD

**Issue by chgibb** [chgibb/PHAT#916]

**Comment by RobJackson28** [comment]
Was SNP track visualization still available in the latest version?

**Comment by chgibb** [comment]
I don't believe it's made it into the UI overhaul, no. I've been a bad steward lately. Do you need it urgently?

**Comment by RobJackson28** [comment]
Not urgent, but I think it's a nice feature to include in the new UI if possible.
